### ISSUE TITLE
About nuscenes training details

### ISSUE THREAD

**Issue by CHANGgreenEVER** [vita-epfl/UniTraj#7]
Thank you for your work! Can you provide the hyperparameters needed to train nuscenes dataset using MTR? For example, learning rate, epoch, attenuation strategy, etc., I directly used MTR to find poor results in the hyperparameters of waymo dataset.

**Comment by Alan-LanFeng** [comment]
past_len: 21 # history trajectory length, 2.1s
future_len: 60 # future trajectory length, 6s
object_type: [ 'VEHICLE'] # object types included in the training set
line_type: [ 'lane','road_edge','road_line'] # line type to be considered in the input
masked_attributes: [ 'z_axis' ] # attributes to be masked in the input
trajectory_sample_interval: 1 # sample interval for the trajectory
only_train_on_ego: False # only train on AV
center_offset_of_map: [ 30.0, 0.0 ] # center offset of the map
use_cache: False # use cache for data loading
overwrite_cache: False # overwrite existing cache
store_data_in_memory: False # store data in memory


(optional) git checkout -b "mtr_ldb_ckpt" b23eacdefc58ce260ffbd65c38d2e6baa4d10d89 The commit for leaderboard ckpt

**Comment by CHANGgreenEVER** [comment]
Thank you so much! Can you share the learning rate, epoch, and decay strategy?

**Comment by Alan-LanFeng** [comment]
They are the same as in the default config

**Comment by CHANGgreenEVER** [comment]
Thank you so much!

**Comment by CHANGgreenEVER** [comment]
In the training of nuscenes V1.0, may I ask how much max_data_num in cfg: [ 1000 ] should be set?

**Comment by Alan-LanFeng** [comment]
You can set a larger number such as 10000000 to ensure that all data is used

**Comment by CHANGgreenEVER** [comment]
Thank you so much! Your work is a milestone in the motion prediction task.
